app/templates/direct-award/_saved-search-not-locked-project-temp-message.html

```diff
@@ -0,0 +1,22 @@
+{% if not project.lockedAt and following_framework %}
+  <div class="column-one-third">
+    {% if temporary_message_status == NOT_LOCKED_PRE_LIVE %}
+      {% set temp_heading =  '{}'.format(following_framework.name)|nbsp + ' services will replace existing services on {}'.format(following_framework.frameworkLiveAtUTC|dateformat) %}
```

Comment:
We have a dmutils.dates helper that converts framework dates into their most common formats, so we could use that Python side rather than format here. Just call `update_framework_with_formatted_dates(following_framework)` and then reference it as `following_framework.frameworkLiveAt` in the template.

Optional change.

Reply:
I'm gonna leave as is, as the template use both `dateformat` and `shortdateformat`